lib/src/tooltip_widget.dart

```diff
@@ -518,9 +676,41 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
                     ),
                     color: Colors.transparent,
                     child: Center(
-                      child: MeasureSize(
-                        onSizeChange: onSizeChange,
-                        child: widget.container,
+                      child: Column(
+                        children: [
+                          Stack(
+                            children: [
+                              // This widget is used for calculation of the action
+                              // widget size and it will be removed once the size
+                              // is calculated
+                              // We have kept it in colum because if we put is
+                              // outside in the stack then it will take whole
+                              // screen size and width calculation will fail
+                              if (isSizeRecalculating) _getOffstageActionWidget,
```

Comment:
Yes this Widget will stay offStage and will be used for only one frame to calculate size of action and then it will be discarded so this is correct place for this.